src/Evolution/Executables/GrMhd/ValenciaDivClean/CMakeLists.txt

```diff
@@ -44,122 +43,27 @@ function(add_grmhd_executable SUFFIX INITIAL_DATA INTERPOLATION_TARGET)
   target_compile_definitions(
     ${EXECUTABLE}
     PRIVATE
-    INITIAL_DATA=${INITIAL_DATA}
     INTERPOLATION_TARGET=${INTERPOLATION_TARGET}
+    USE_PARAMETRIZED_DELEPTONIZATION=${USE_PARAMETRIZED_DELEPTONIZATION}
     )
   target_link_libraries(${EXECUTABLE} PRIVATE ${LIBS_TO_LINK})
 endfunction()
 
 add_grmhd_executable(
-  FishboneMoncriefDisk
-  RelativisticEuler::Solutions::FishboneMoncriefDisk
+  WithHorizon
   KerrHorizon
-  )
-
-add_grmhd_executable(
-  TovStar
-  RelativisticEuler::Solutions::TovStar
-  CenterOfStar
-  )
+  false
+)
 
 add_grmhd_executable(
-  AlfvenWave
-  grmhd::Solutions::AlfvenWave
   ""
-  )
-
-add_grmhd_executable(
-  BondiMichel
-  grmhd::Solutions::BondiMichel
   ""
-  )
-
-add_grmhd_executable(
-  KomissarovShock
-  grmhd::Solutions::KomissarovShock
-  ""
-  )
-
-add_grmhd_executable(
-  SmoothFlow
-  grmhd::Solutions::SmoothFlow
-  ""
-  )
-
-add_grmhd_executable(
-  BondiHoyleAccretion
-  grmhd::AnalyticData::BondiHoyleAccretion
-  ""
-  )
-
-add_grmhd_executable(
-  BlastWave
-  grmhd::AnalyticData::BlastWave
-  ""
-  )
-
-add_grmhd_executable(
-  CcsnCollapse
-  grmhd::AnalyticData::CcsnCollapse
-  ""
-  )
-
-add_grmhd_executable(
-  KhInstability
-  grmhd::AnalyticData::KhInstability
-  ""
-  )
+  false
+)
 
+# Core-collpase SN uses parametrized deleptonization
 add_grmhd_executable(
-  MagneticFieldLoop
-  grmhd::AnalyticData::MagneticFieldLoop
+  CCSN
```

Comment:
@pajkosmi do you have a wish for the name of your executable? Suggestions: `EvolveCcsn`, `EvolveCoreCollapse`, `EvolveSupernova`, `MikesExecutable`

Reply:
`EvolveCoreCollapseSupernova` is my vote.  It's a tad long but the most precise.

Reply:
Convention from other executables would be `EvolveGhValenciaDivCleanCcsn`. For example, we have `EvolveGhValenciaDivCleanBns`. I'd prefer to follow conventions since that makes finding executables easier.

Reply:
Yeah I do think it's important for GhValenciaDivClean to be in there somehow, current consensus would be `EvolveGhValenciaDivCleanCoreCollapseSupernova` but that's a bit long.

Reply:
Maybe shorten to `EvolveGhmhdCcsn` (and also `EvolveGhmhdBns` etc)? That's a lot of abbreviations but it's short :)

Reply:
I think `EvolveGhValenciaDivCleanCcsn` is fine. Again, we have `EvolveGhValenciaDivCleanBns` already, which I think has a reasonable balance of length and description. There are other formulations of MHD that one could try to use, so I don't like using `mhd`.